### ISSUE TITLE
switch from Python to PHP 

### ISSUE THREAD

**Issue by emrox** [emrox/photonet#1]
not everyone can install a Python-Server on a own server.
PHP is wide spread and can be copied to most webspace packages, so the installation should be easier. Even PHP is not the best consistent programing language, with a little help from frameworks it should be possible to create a good readable and maintainable code base.

Todo:
- find a good and small framework which can be used on most of PHP enabled webspaces
- rewrite the python stack to php

**Comment by emrox** [comment]
[Lumen](http://lumen.laravel.com/) macht auf den ersten Blick einen guten Eindruck. Etwas mehr als Micro-Framework, aber dennoch nicht gleich der grosse Hammer

**Comment by JanAhrens** [comment]
I second that :+1: 

I did a small project with [Slim](http://www.slimframework.com/) a while ago. Maybe that's a good starting point.

We should also think about providing self contained zip-files that are easy to install, because they contain all dependent libraries. For development we can still use a dependency management tool like [Composer](https://getcomposer.org/) as long as we provide such release bundles.

**Comment by emrox** [comment]
yes, slim would be a great choice. 2nd candidate will be [Lumen](http://lumen.laravel.com) which is very new and from the laravel team.

bad thing on slim is that the 3.0 version is on work and will released soon (or at least in reachable future) - but since our project is so small ATM it should not be so hard to rewrite it after the 3.0 is released.

**Comment by emrox** [comment]
and zip-files should be our main release format (no non-developer can use composer)

**Comment by Umorrian** [comment]
I'd recommend using something like [Lazer-Database](https://github.com/Greg0/Lazer-Database) for data storage. That way there is no need to configure an actual database. Additionally, if the images are stored within the installation folder, everything can be zipped and moved to a diffrent php server easily.

I'd rather vote for lumen then slim, just because I'm somewhat familiar with laravel.

**Comment by emrox** [comment]
:+1:  for Lazer-Database

i'm more into slim because for the default setup the starting index.php is on the root folder and not within a extra public folder. Makes it easier for just-copy-everything-to-your-webspace-installations. If this can also be done easily within lumen i'm unsure what to use.
